src/test/feature-debug.spec.ts

```diff
@@ -29,13 +31,24 @@ describe("feature-debug", () => {
             res.indexOf("=== DEBUG LOGS ===") + 19,
             res.indexOf("=== VM LOGS ===") - 2,
         );
-        expect(debugLogs).toStrictEqual(`stack(2 values) : 10000000000 () 
+
+        const filePath = path.normalize("src/test/features/debug.tact");
```

Comment:
Will this work on Windows? Perhaps we need to fix it in the tests

Reply:
Oh, sorry, was reviewing it from my phone and it seemed like the fix was applied in the main logic :)